plot_Pareto_bokeh added to plots.py

abbl-DTI opened the pull request:
The new function creates an interactive plot with the bokeh module. There are several options for the output.
This still needs to be tested with different IDEs e.g. spyder.
This related to issue #84

abbl-DTI committed:
plot_Pareto_bokeh added to plots.py

The new function creates an interactive plot with the bokeh module. There are several options for the output.
This still needs to be tested with different IDEs e.g. spyder.

abbl-DTI committed:
Change of bokeh import

The import of different bokeh features are changed to be less generic.

dk-teknologisk-mon committed:
Cleans up the documentation

dk-teknologisk-mon committed:
Clean-up of formatting according to PEP 8

dk-teknologisk-mon committed:
Fixes error in keyword

abbl-DTI committed:
Merge branch 'novonordisk-research:develop' into Bokeh_Pareto_plot

abbl-DTI committed:
Bokeh json object output added

The option of exporting the Pareto plot rendered with Bokeh as json-item is added.

abbl-DTI commented:
This feature is ready to be merged to the develop branch.

sqbl commented:
Looking good. 
Can we add a test to ensure that return_data returns something with the right format? (It might be tricky to ensure a consistent numerical return, but we should be able to get the same structure in the return everytime)
And currently, Bokeh is optional in installation. Should we make is mandatory to install?

abbl-DTI committed:
plot_Pareto_bokeh updated and test_plot.py created

Minor changes have been added to the plot call 'plot_Pareto_bokeh' in order to serve a json-object for plotting with bokehjs.
pytest for the 'plot_Pareto_bokeh' call have been created.
